flutter_appauth/README.md

```diff
@@ -5,7 +5,10 @@
 
 A Flutter bridge for AppAuth (https://appauth.io) used authenticating and authorizing users. Note that AppAuth also supports the PKCE extension that is required some providers so this plugin should work with them.
 
-If you're using Auth0, there is a detailed post on the Auth0 blog on how you can use this plugin with Auth0 that you can check out by opening [this link](https://auth0.com/blog/get-started-with-flutter-authentication/).
+Provider tutorials:
```

Comment:
Could this be rephrased? This is to avoid confusion with the [provider](https://pub.dev/packages/provider) package in case readers think they will be seeing how to see this plugin together with the package. Not sure what this could options there are here but may need to longer to make it clearer. Maybe "Identity provider tutorials" is enough or something like "Tutorials from identity providers". Happy to take suggestions here.

Also as it's a section in itself, using markdown to indicate this would help call that out more

Reply:
Split out the section, made it clear it wasn't related to the "provider" package. Thanks for the feedback!